integration-tests/lra/src/test/java/org/apache/camel/quarkus/component/lra/it/LraTestResource.java

```diff
@@ -28,7 +28,7 @@
 public class LraTestResource implements QuarkusTestResourceLifecycleManager {
 
     private static final Integer LRA_PORT = AvailablePortFinder.getNextAvailable();
-    private static final String LRA_IMAGE = "jbosstm/lra-coordinator:5.12.0.Final";
+    private static final String LRA_IMAGE = "quay.io/jbosstm/lra-coordinator";
```

Comment:
Is there a tag that we can pin to?

Reply:
Good point! I think it could be `5.13.1.Final-2.16.4.Final` which is composed by `narayana-version` and `quarkus-version`. Anyway, I can check it with narayana team.